sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -2646,17 +2647,16 @@ def hack_iter_next_mock_error(self):
                 self._open()
                 # when trying to receive the second message (execution_times is 1), raising LinkDetach error to mock 10 mins idle timeout
                 if self.execution_times == 1:
-                    from uamqp.errors import LinkDetach
-                    from uamqp.constants import ErrorCodes
+                    from azure.servicebus._pyamqp.error import ErrorCondition, AMQPConnectionError
                     self.execution_times += 1
                     self.error_raised = True
-                    raise LinkDetach(ErrorCodes.LinkDetachForced)
+                    raise AMQPConnectionError(condition=ErrorCondition.LinkDetachForced)
```

Comment:
Don't we have a LinkDetach error in pyamqp?

Reply:
(Sorry missed this one) I have only seen an error condition for linkDetached

Reply:
We're both half right - there's no LinkDetactError, but we do have a link error ;)
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/error.py#L257